extensions/extensions-docker/aissemble-airflow/src/main/resources/docker/Dockerfile

```diff
@@ -1,7 +1,7 @@
 # Script for creating base Airflow Docker image
 FROM apache/airflow:2.7.0-python3.11
 
-LABEL org.opencontainers.image.source = "https://github.com/boozallen/aissemble"
+LABEL org.opencontainers.image.source="https://github.com/boozallen/aissemble"
```

Comment:
I: This and the `ENV` format change are to address a deprecation warning that was output by the docker-maven-plugin